torch/_torch_docs.py

```diff
@@ -3091,7 +3091,23 @@ def merge_dicts(*dicts):
            r"""
 flatten(input, start_dim=0, end_dim=-1) -> Tensor
 
-Flattens a contiguous range of dims in a tensor.
+Flattens :attr:`input`, or if :attr:`start_dim` or :attr:`end_dim` are specified, a range of :attr:`input`
```

Comment:
Flattens :attr:\`input\` or, if :attr:\`start_dim\` or :attr:\`end_dim\` are specified, then this flattens the dimensions of :attr:\`input\` between :attr:\`start_dim\` and :attr:\`end_dim\`, inclusive.

Reply:
"a range of input dimensions" -> "flattens dimensions in input starting with start_dim and ending with end_dim."